L1Trigger/L1THGCal/interface/be_algorithms/HGCalClusteringImpl.h

```diff
@@ -1,9 +1,6 @@
 #ifndef __L1Trigger_L1THGCal_HGCalClusteringImpl_h__
 #define __L1Trigger_L1THGCal_HGCalClusteringImpl_h__
 
-#include <array> 
```

Comment:
`std::array` is present in the class interface (`triggerCellReshuffling()` parameters), so it should be kept here.